src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -77,6 +77,62 @@ internal static ParseResult ParseScript(string script, ILanguageSettings loc = n
             return new ParseResult(parsetree, errors, errors?.Any() ?? false, parser._comments, parser._before, parser._after);
         }
 
+        internal static ParseFormulasResult ParseFormulasScript(string script, ILanguageSettings loc = null, Flags flags = Flags.None)
```

Comment:
As I see it, this is a meta-language on top of the expression language. Might be worth having it in a separate file, i.e. NamedFormulaParser.cs